package.json

```
@@ -1,9 +1,13 @@
 {
+  "name": "@node-core/api-docs-tooling",
   "scripts": {
     "lint": "eslint .",
     "format": "prettier --write .",
     "prepare": "husky"
   },
+  "bin": {
+    "api-docs-tooling": "./bin/cli.mjs"
```

```suggestion
    "api-docs-tooling": "./bin/cli.mjs"
```

What happens if you run `npx @node-core/api-docs-tooling` (?) and with `npx api-docs-tooling"` (?)

---

Outputs;

```sh
npx @node-core/api-docs-tooling
sh: api-docs-tooling: command not found
```

```sh
npx api-docs-tooling           
error: required option '-i, --input <patterns...>' not specified
```